packages/devtools_app/lib/src/screens/debugger/codeview.dart

```diff
@@ -249,7 +262,10 @@ class _CodeViewState extends State<CodeView>
     final scriptSource = parsedScript?.script.source;
     if (script != null && scriptSource != null) {
       if (scriptSource.length < 500000) {
-        final highlighted = script.highlighter.highlight(context);
+        final highlighted = script.highlighter.highlight(
+          context,
+          lineRange: widget.lineRange,
```

Comment:
Maybe add a TODO to switch from having lineRange to highlightedLineRange where text outside the highlighted line range has a greyed background and selection is auto scrolled to the highlighted range. that would be great for the inspector use case. Note we already scroll to lines on hitting a breakpoint which could interact with this in tricky ways.